gocourse05/server.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+
+	"gocourse05/camera"
+)
+
+const serverUrl = "http://remote.animals.control"
+
+type Memory []camera.ProcessedData
+
+func saveProcessedData(p camera.Processor, m *Memory) error {
```

Comment:
Напевно ось так має бути правильніше:
```suggestion
func (m *Memory) saveProcessedData(p camera.Processor) error {
```

В пам'яті є метод "зберегти".